infra/docker_repo_manager_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test the functionality of the DockerRepoManager class
+The will consist of the following functional tests
+  1. Test the construction and paring of a Docker image and a git repo
+  2. Test that the class can create a docker image with a specific commit
+"""
+
+from DockerRepoManager import DockerRepoManager 
+import unittest
```

Comment:
`unittest` import should come first since it is a Python standard library import (see http://google.github.io/styleguide/pyguide.html#313-imports-formatting)